flocker/node/functional/test_docker.py

```diff
@@ -334,11 +366,13 @@ def test_pull_image_if_necessary(self):
         """
         The Docker image is pulled if it is unavailable locally.
         """
-        image = u"busybox:latest"
+        # Use an image that isn't likely to be in use by anything, since
+        # it's old, and isn't used by other tests:
+        image = u"busybox:ubuntu-12.04"
```

Comment:
Jean-Paul has better fix in #1281, mostly I just wanted to make it less likely so I could run my own tests better. Main goal was fixing 404s.